pkg/cpu/cpu_linux.go

```diff
@@ -112,6 +113,9 @@ func processorsGet(ctx *context.Context) []*Processor {
 	}
 	res := []*Processor{}
 	for _, p := range procs {
+		for _, c := range p.Cores {
+			sort.Ints(c.LogicalProcessors)
```

Comment:
I can't point my finger at anything but this looks suspicious. Let's monitor CI after we merge this PR. Not stopping because I can't back my gut feeling with any hard data.

Reply:
Yeah, it's just sorting the logical processor indexes to avoid situations like [here](https://github.com/jaypipes/ghw/issues/345#issuecomment-1620274077) where I thought there was a bug because the logical processors listed in a non-sorted way :)